api/types.proto

```diff
@@ -280,6 +280,23 @@ message ContainerStatus {
 
 	int32 pid = 2 [(gogoproto.customname) = "PID"];
 	int32 exit_code = 3;
+
+	HealthStatus health = 4;
+}
+
+// HealthStatus stores information about the container's healthcheck results.
+message HealthStatus {
+	string status = 1;
+	int32 failing_streak = 2;
```

Comment:
I don't think we should choose option 4, because then we lose out on the extra info that's part of the `HealthcheckResult` logs.